cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/containers/NotifyContextRequestNGSIv2.java

```diff
@@ -0,0 +1,313 @@
+/**
+ * Copyright 2014-2017 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of fiware-cygnus (FIWARE project).
+ *
+ * fiware-cygnus is free software: you can redistribute it and/or modify it under the terms of the GNU Affero
+ * General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ * fiware-cygnus is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
+ * implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with fiware-cygnus. If not, see
+ * http://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
+ */
+package com.telefonica.iot.cygnus.containers;
+
+import com.google.gson.JsonElement;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+/**
+ * The type Notify context request ngs iv 2.
```

Comment:
```suggestion
 * The type Notify context request NGSIv2.
```

Reply:
Fixed in[9b6d90b](https://github.com/telefonicaid/fiware-cygnus/pull/1762/commits/9b6d90ba354adf7cd42e45fbe9e335df9650e200)